### ISSUE TITLE
Feature Request - allow specifying API_KEYS via docker secret

### ISSUE THREAD

**Issue by mcgodfrey** [supertokens/supertokens-docker-postgresql#41]
It is currently possible to specify the postgresql password as a docker secret using POSTGRESQL_PASSWORD_FILE environment variable.

It would be nice to be able to also specify API_KEYS in a similar way.
ie. to add an environment variable called API_KEYS_FILE which would open the specified file and set API_KEYS to the contents.

I think this should just be a simple addition to the docker-entrypoint.sh script, similar to how POSTGRES_PASSWORD_FILE is handled.
eg.

```
if [ ! -z $API_KEYS_FILE ]
then
    API_KEYS=$(cat "$API_KEYS_FILE")
    export API_KEYS
fi
```

Thanks!

**Comment by rishabhpoddar** [comment]
> It is currently possible to specify the postgresql password as a docker secret using POSTGRESQL_PASSWORD_FILE environment variable.

I don't think we support this. But, we can add this to our docker entrypoint when we have time. Alternatively, you could make a PR for it :)
